src/main/scala/org/hammerlab/guacamole/commands/GermlineThresholdCaller.scala

```diff
@@ -115,9 +115,8 @@ object GermlineThreshold {
           }
 
           sortedAlleles match {
-            /* If no alleles are above our threshold, we emit a NoCall variant with the reference allele
-           * as the variant allele.
-           */
+            // If no alleles are above our threshold, we emit a NoCall variant with the reference allele
+            // as the variant allele.
```

Comment:
It was mis-indented so I was changing it anyway. I've always used `//` for inline comments and `/* .. */` exclusively for scaladoc/javadoc comments. Do you guys have any conventions around this?